### ISSUE TITLE
Problem: development process is not clear

### ISSUE THREAD

**Issue by zolotokrylin** [holdex/pr-time-tracker#359]
@georgeciubotaru to confirm, I can proceed with this way?
and actually I have one question, as this migration is related to each other, I don't think I can separate it to multiple PRs that can be merged to main.
So how should I approach this? one big PR is okay? or can I have multiple PRs that branches from the other PR? e.g. feat/v3 <- feat/webhook <- feat/something-else

I still have one question, which approach should I use for these PRs?

_Originally posted by @teodorus-nathaniel in https://github.com/holdex/pr-time-tracker/issues/340#issuecomment-2565167407_
            
# Blocks
- https://github.com/holdex/pr-time-tracker/issues/360

**Comment by zolotokrylin** [comment]
@teodorus-nathaniel I suggest forking the repo into a new one (effectively create a new one, as forking might not provide any benefits).

@georgeciubotaru, your thoughts?

**Comment by georgeciubotaru** [comment]
@zolotokrylin, I agree that we need to move all trigger dev functionality into a separate repo and have only the UI in this one.

**Comment by georgeciubotaru** [comment]
@teodorus-nathaniel As mentioned here: https://github.com/holdex/pr-time-tracker/issues/360#issuecomment-2565914855

We can take 1 webhook and develop it in 1 PR:

1 PR:
- Add new webhook + handle the task (we can start with only the webhook and then handle and connect it when is fully done)
- disable the existing job

**Comment by teodorus-nathaniel** [comment]
hi @georgeciubotaru sorry for the late response, I still have some questions regarding this:
1. fork the existing repo or should I make a new sveltekit project that consist only for trigger.dev webhook?
2. so you mean for the first PR, I should only migrate 1 task, and disable the webhook for that specific task that I migrated in the old project, and migrate the others after the first one is running properly?

**Comment by georgeciubotaru** [comment]
1. new project
2. yes

@teodorus-nathaniel

**Comment by teodorus-nathaniel** [comment]
one more question, is it okay if I use next js for this? @zolotokrylin @georgeciubotaru 

issue is because svelte has this env that can be imported using `$env/static`, but to create jobs, this is not supported, I can use process.env, but it will cause multiple ways to import env, which may be confusing.

**Comment by georgeciubotaru** [comment]
@teodorus-nathaniel don't get why you can't use it, but feel free to use next js as mostly we don't have UI

**Comment by teodorus-nathaniel** [comment]
@georgeciubotaru I can't use the svelte env because the tasks will be deployed on the trigger dev server, which doesn't use svelte-kit to start the project, that's why in the trigger.dev we need to put the necessary envs too. In other words, it will not use the .env file from the pr-time-tracker server
![image](https://github.com/user-attachments/assets/22808bad-f1d2-48a6-b884-46c35b818efb)
